apps/settings/js/modules/sim_security.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * This is a simple model that can help us mimic the status change event
+ * when enabling / disabling simcard.
+ *
+ * @module SimSecurity
+ */
+define(function(require) {
+  'use strict';
+
+  var Module = require('modules/base/module');
+  var SettingsUtils = require('modules/settings_utils');
+  var EventEmitter = require('modules/base/event_emitter');
+
+  var APIS = [
+    'unlockCardLock',
+    'setCardLock',
+    'updateContact',
+    'getCardLock',
+    'getCardLockRetryCount'
+  ];
+
+  var EVENT_CHANGE_APIS = [
+    'unlockCardLock',
+    'setCardLock'
+  ];
+
+  var SimSecurity = Module.create(function() {
+    this.super(EventEmitter).call(this, [
+      'sim-security-pin-enabled',
```

Comment:
I think we can simply use `pin-enabled`, `pin-disabled`... etc without the `sim-security` prefix because it is pretty clear that the events are from the module `SimSecurity` when the code is like:
```
SimSecurity.addEventListener('pin-disabled', function() {});
```